document_page/views/document_page.xml

```diff
@@ -1,309 +1,309 @@
 <?xml version="1.0"?>
 <odoo>
-    <data>
-       <menuitem name="Pages"
-                 id="menu_wiki"
-                 parent="knowledge.menu_document"
-                 sequence="20" />
 
-        <!-- wiki tree view -->
-        <record id="view_wiki_tree_children" model="ir.ui.view">
-            <field name="name">document.page.tree</field>
-            <field name="model">document.page</field>
-            <field name="field_parent">child_ids</field>
-            <field name="priority">100</field>
-            <field name="arch" type="xml">
-                <tree string="Document Page">
-                    <field name="name"/>
-                    <field name="write_uid"/>
-                    <field name="write_date"/>
-                </tree>
-            </field>
-        </record>
+    <menuitem name="Pages"
```

Comment:
This is the top menu ("App") name.
Should we rather name it "Documents" or "Knowledge" ?